<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckOption</class>
 <widget class="QDialog" name="CheckOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog[objectName=&quot;CheckOption&quot;]
{
border-image: url(:/ui/Input/Input_Frame.png);
background-color:transparent;
border-radius:10px;
}</string>
  </property>
  <widget class="QLabel" name="label_ID_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>113</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/lineaedit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>89</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-radius:10px;
color:green;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>145</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-image: url(:/ui/Input/cancel.png);
font-size:30px bold;

border-radius:10px;
}

QPushButton:pressed {
	background-image: url(:/ui/Input/canel_select.png);
font-size:30px bold;

border-radius:10px;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>43</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/lineaedit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>145</y>
     <width>106</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-image: url(:/ui/Input/ok.png);
font-size:30px bold;

border-radius:20px;
}

QPushButton:pressed {
	
	background-image: url(:/ui/Input/ok_select.png);
font-size:30px bold;

border-radius:20px;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Bach">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>53</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-radius:10px;
color:green;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Gender">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>8</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/lineaedit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Operator">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>123</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-radius:10px;
color:green;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>53</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-radius:10px;
color:green;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_gender">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>18</y>
     <width>125</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bach">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>43</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/lineaedit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Age">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>87</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-radius:10px;
color:green;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>9</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/lineaedit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>77</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/lineaedit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>19</y>
     <width>125</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Age">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>77</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/lineaedit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>14</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/type.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>48</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/id.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>13</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/gender.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>51</y>
     <width>79</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/bach.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>82</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/age.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>117</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/operator.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>82</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ui/Input/name.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_Type</zorder>
  <zorder>label_ID_2</zorder>
  <zorder>pushButton_Cancel</zorder>
  <zorder>label_ID</zorder>
  <zorder>pushButton_OK</zorder>
  <zorder>label_Gender</zorder>
  <zorder>label_Bach</zorder>
  <zorder>label_Name</zorder>
  <zorder>label_Age</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>lineEdit_Operator</zorder>
  <zorder>lineEdit_Name</zorder>
  <zorder>lineEdit_ID</zorder>
  <zorder>lineEdit_Bach</zorder>
  <zorder>lineEdit_Age</zorder>
  <zorder>comboBox</zorder>
  <zorder>comboBox_gender</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
